evm/v0.5/contracts/dev/Owned.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity 0.5.0;
+
+/**
+ * @title The Owned contract
+ * @notice A contract with helpers for basic contract ownership.
+ */
+contract Owned {
+
+  address public owner;
+  address private pendingOwner;
+
+  event OwnershipTransferRequested(
+    address indexed to,
+    address from
+  );
+  event OwnershipTransfered(
```

Comment:
Just to follow convention of [OpenZepplin Ownable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/ownership/Ownable.sol#L16), can we switch the order of these params?